src/mmw/js/src/analyze/views.js

```diff
@@ -60,9 +61,13 @@ var AnalyzeWindow = Marionette.LayoutView.extend({
     },
 
     showHeaderRegion: function() {
-        this.headerRegion.show(new HeaderView({
-            model: new models.AnalyzeModel({
-                shape: this.model.get('area_of_interest')
+        this.headerRegion.show(new coreViews.AreaOfInterestView({
+            App: App,
+            model: new coreModels.AreaOfInterestModel({
+                shape: this.model.get('area_of_interest'),
+                canGoBack: true,
```

Comment:
Looks like we decided to use it for all models: https://github.com/WikiWatershed/model-my-watershed/issues/237. The comment thread is gone, unfortunately. I'd still vote for that.